notebooks/10_Working_with_Large_Datasets.ipynb

```diff
@@ -190,7 +189,6 @@
    "outputs": [],
    "source": [
     "%opts RGB [xaxis=None yaxis=None]\n",
-    "import geoviews as gv\n",
     "from bokeh.models import WMTSTileSource\n",
     "url = 'https://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{Z}/{Y}/{X}.jpg'\n",
     "wmts = WMTSTileSource(url=url)\n",
```

Comment:
This and the import above could be removed, as you can now just pass the ``url`` to ``gv.WMTS(url)``.

Reply:
Ignore this if you want to exclusively address flakes in this PR.